vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Installing and Using Packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Installing and Using Packages on Linux
+
+## Description
+
+In this tutorial, you will learn how to use vcpkg to set up a new "Hello World" project using the `fmt` library, manage dependencies, build, and run your C++ application. 
+
+## Setup vcpkg
+
+1. **Clone the vcpkg repository**
+
+   Downloads the vcpkg registry.
+
+    ```bash
+    git clone https://github.com/microsoft/vcpkg.git
+    ```
+
+   vcpkg's curated registry refers to a set of libraries that have been vetted and packaged to work smoothly with the vcpkg package management system. These libraries are usually open-source projects from various domains, and they are included in vcpkg's main GitHub repository. Each package or "port" in the curated registry typically contains:
```

Comment:
```suggestion
The curated registry is a set of libraries that have been vetted and packaged to work smoothly with vcpkg. vcpkg uses "ports" to package open-source projects and make them available to install. Ports typically contain: 
```

Some information here seems irrelevant or redundant, since this is a tutorial, we should avoid overwhelming readers.